test/integration/config-ci.js

```diff
@@ -0,0 +1,16 @@
+// this file is for use in CircleCI continuous integration environment
+module.exports = {
+    seleniumServerURL: {
+        hostname    : 'ondemand.saucelabs.com',
+        port        : 80,
+    },
+    driverCapabilities: {
+        platform            : 'Windows 7',
+        'tunnel-identifier' : 'circle-' + process.env.CIRCLE_PROJECT_USERNAME + '-' + process.env.CIRCLE_PROJECT_REPONAME + '-' + process.env.CIRCLE_BUILD_NUM + '-' + process.env.CIRCLE_NODE_INDEX
+    },
+    tags        : [ 'circle-ci', '#' + process.env.CIRCLE_BUILD_NUM ],
+    views       : [ 'Verbose', 'SauceLabs' ],
```

Comment:
No, due to [precedence rules](https://github.com/MattiSG/Node-ConfigLoader#paths).